### PR TITLE
Features/create homepage

### PR DESCRIPTION
**Pull request by GrandSuccess87**
Create homepage story is completed

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,14 +1,32 @@
...
+it('renders a new page after start button is click', () => {
+  const wrapper = shallow(<Main />);
+  wrapper.find('a').simulate('click');
+  const linkToGame = wrapper.find('.game-page');
+  expect(linkToGame.length).toEqual(1);
+});
+
 it('asserts false', () => {
   expect(false).toEqual(false);
 });
```
Comment (soft-circles) [inline src/App.test.js]: you can delete this last test

File: src/components/gameGomponent/gameComponent.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+
+class Game extends Component {
+   render() {
+       return (
+           <div>
+               <div className="Game-header">
```
Comment (soft-circles) [inline src/components/gameGomponent/gameComponent.js]: Can we extract this into a `GameHeader` component?

File: src/App.js [hunk 3]
```diff
@@ -1,19 +1,19 @@
 import React, { Component } from 'react';
 import './App.css';
 import Main from "./components/mainComponent/mainComponent";
-// import Game from "./components/gameComponent/gameComponent";
+import Game from "./components/gameGomponent/gameComponent";
+// import { BrowserRouter, Route, Link, Switch } from "react-router-dom";
```
Comment (soft-circles) [inline src/App.js]: delete unused `import` line

File: src/App.test.js (hunk 1 above)
Comment (GrandSuccess87) [reply src/App.test.js]: Test deleted

File: src/components/gameGomponent/gameComponent.js (hunk 2 above)
Comment (GrandSuccess87) [reply src/components/gameGomponent/gameComponent.js]: Working on it now

File: src/App.js (hunk 3 above)
Comment (GrandSuccess87) [reply src/App.js]: Unused `import` line deleted

File: src/components/gameGomponent/gameComponent.js (hunk 2 above)
Comment (GrandSuccess87) [reply src/components/gameGomponent/gameComponent.js]: Extracted into a GameHeader Component

**Review by soft-circles — CHANGES_REQUESTED** [review]
Please see comments

**Comment by GrandSuccess87** [comment]
I see it says that there are conflicting files. My thought is because master is not currently updated in real time with the files I have added in the features/createHomepage branch and is therefore reading this as a merge conflict. This now builds successfully in Travis CI.

**Review by soft-circles — APPROVED** [review]

**Pull request merged**